src/Microsoft.ML.GenAI.Core/Microsoft.ML.GenAI.Core.csproj

```diff
@@ -9,10 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
-    <PackageReference Include="TorchSharp" Version="0.102.5" />
-    <PackageReference Include="libtorch-cpu-win-x64" Version="2.2.1.1" Condition="$([MSBuild]::IsOSPlatform('Windows'))" PrivateAssets="all" />
-    <PackageReference Include="libtorch-cpu-linux-x64" Version="2.2.1.1" Condition="$([MSBuild]::IsOSPlatform('Linux'))" PrivateAssets="all" />
-    <PackageReference Include="libtorch-cpu-osx-x64" Version="2.2.1.1" Condition="$([MSBuild]::IsOSPlatform('OSX'))" PrivateAssets="all" />
+    <PackageReference Include="TorchSharp" Version="$(TorchSharpVersion)" />
```

Comment:
We should revert this change for disk space reasons. I mean update the version yes, but keep the cpu libs split out by OS

Reply:
This is not torchsharp runtime package, so the size will not be huge

It would be upon user’s preference to choose which torch runtime to install when they consume genai packages

Reply:
Gotcha. Sounds good to me then as is.